islandora_videojs.module

```diff
@@ -64,9 +66,12 @@ function template_preprocess_islandora_videojs(array &$variables) {
   if (isset($params['sources']) && is_array($params['sources'])) {
     $variables['sources'] = $params['sources'];
   }
-  else {
+  elseif (isset($params['url']) && isset($params['mime'])) {
     $variables['sources'][] = array('url' => $params['url'], 'mime' => $params['mime']);
   }
+  else {
+    $variables['sources'] = array();
```

Comment:
I agree with Diego (as a site administrator) on his minor note. May I propose something along the lines of "Video file not found."?